libero/src/main/groovy/com/connexta/libero/Config.groovy

```diff
@@ -3,28 +3,35 @@ package com.connexta.libero
 import static org.ho.yaml.Yaml.load
 
 class Config {
-    Map projectProperties = new HashMap()
+
     String projectDir
     String projectName
-    File configFile
     String sourceRemote
     String destRemote
     String ref
     String destBranch
+
     String startVersion
     String releaseVersion
     String nextVersion
+    String releaseName
+
     String mavenRepo
     Map preProps
     Map postProps
-    String releaseName
+
```

Comment:
Should we consider moving the `Options` class I created into your module and use that in both of our scripts?